### DISCUSSION TITLE
Which 🐶 breed was the favorite in the 🇺🇸 until 2022?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#982] [Galaxy Trivia] [answered]
**Bonus:** What is a group of these 🐕 called?

**Comment by Sriansh-raj** [comment]
The **French Bulldog** was the most popular dog breed in the United States in 2022, overtaking the Labrador Retriever, which had held the top spot for 31 years.

According to LovingFrenchBulldogs.com, a group of French Bulldogs is typically referred to as a "litter"
. However, a Reddit user claims that a group of French Bulldogs is called a "grumble"
Another source, an Instagram post, suggests that a group of French Bulldogs can also be called a "froggle"
It is important to note that these terms are not official and are not widely recognized.

**Reply by ParanoidUser** [reply]
Nice try! Keep it up 👍

**Comment by peterbaccaro** [comment, marked as answer]
The **Labrador Retriever** dog breed was the favorite in the US until 2022.

A group of Labrador Retrievers is commonly referred to as a **pack**.
